watch.go

```diff
@@ -36,6 +37,11 @@ func (e *Executor) watchTasks(calls ...taskfile.Call) error {
 		}()
 	}
 
+	wInt, err := e.Taskfile.ParsedWatchInterval()
+	if err != nil || wInt == 0 {
+		wInt = watchInterval
```

Comment:
The `watchInterval` const in this file should be renamed to `defaultWatchInterval`.

I would then rename the `wInt` variable to `watchInterval`.